drivers/windows/file_access_windows.cpp

```diff
@@ -168,12 +168,10 @@ void FileAccessWindows::close() {
 				close_fail_notify(save_path);
 			}
 
-			ERR_EXPLAIN("Safe save failed. This may be a permissions problem, but also may happen because you are running a paranoid antivirus. If this is the case, please switch to Windows Defender or disable the 'safe save' option in editor settings. This makes it work, but increases the risk of file corruption in a crash.");
+			ERR_FAIL_COND_MSG(rename_error, "Safe save failed. This may be a permissions problem, but also may happen because you are running a paranoid antivirus. If this is the case, please switch to Windows Defender or disable the 'safe save' option in editor settings. This makes it work, but increases the risk of file corruption in a crash.");
```

Comment:
That's wrong, it will return earlier than before, and `save_path` will no longer be reset. The cond check is also redundant.